WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs

```diff
@@ -0,0 +1,19 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Hotfix;
+
+namespace WowPacketParserModule.V7_0_3_22248.Hotfix
+{
+    [HotfixStructure(DB2Hash.PvpTalent, HasIndexInData = false)]
+    public class PvpTalentEntry
+    {
+        public uint SpellID { get; set; }
+        public uint OverridesSpellID { get; set; }
+        public string Description { get; set; }
+        public uint TierID { get; set; }
+        public uint ColumnIndex { get; set; }
+        public uint Flags { get; set; }
```

Comment:
Woops, this is a guessed name. Nothing more than that I am afraid. I should have mentioned it. Reason why I guessed it was due to the similarities with Talent.db2.